codec/packer.go

```diff
@@ -155,3 +155,5 @@ func (p *Packer) addErr(err error) {
 func (p *Packer) Offset() int {
 	return p.Packer.Offset
 }
+
+type Bytes []byte
```

Comment:
Why do we need to define this type?

Reply:
Yes, we do. Otherwise reflect treats []byte as []uint8